.github/workflows/docs-check.yml

```diff
@@ -36,12 +38,6 @@ jobs:
         with:
           python-version: 3.9
 
-      - uses: DavidAnson/markdownlint-cli2-action@05f32210e84442804257b2a6f20b273450ec8265 # v19.1.0
-        with:
-          globs: |
-            Documentation/**/*.md
-            !Documentation/Helm-Charts
```

Comment:
can we keep this? and have the same tool locally installed...I really like this action yaml as this are fast in CI and for me who mostly depends on GitHub CI and does not run many commands locally so I'd like to keep this.

Also, if others agree to go with your approach feel free to skip my comment I don't want to impose my way of development.

Reply:
One of the exceptionally nice things about the github workflows is that dependabot keeps them up to date automatically. With the changes as noted here, no one is going to be managing the markdownlint-cli2 version in the makefile, and it will stagnate.

This action is critical for ensuring our documentation renders correctly on the rook.io website, and don't really want to risk correctness or long-term maintainability here.

Given the issue you mentioned that this has to be `npm` installed on linux systems, and given the issue with keeping versions up to date, I'm inclined to go with Subham here. I think it's in the better interest of the project to keep this as a GH action.

Reply:
Good points. I will therefore remove/undo my change to the workflow file, thereby kepping it to use the action.

Reply:
done